pages.it/common/alacritty.md

```diff
@@ -0,0 +1,24 @@
+# alacritty
+
+> Multipiattaforma, GPU-accelerato emulatore di terminale.
+> Maggiori Informazioni: <https://github.com/alacritty/alacritty>.
+
+- Apri un nuovo caso di Alacritty:
+
+`alacritty`
+
+- Esegui in una posizione specifica:
+
+`alacritty --working-directory {{path/to/directory}}`
```

Comment:
Please translate the arguments for get this easier for non-English speakers.
Checking another `it`:
```suggestion
`alacritty --working-directory {{percorso/a/cartella}}`
```
is this OK?